src/api/spec/support/capybara.rb

```diff
@@ -2,6 +2,7 @@
 
 Capybara.default_max_wait_time = 6
 Capybara.save_path = Rails.root.join('tmp', 'capybara')
+Capybara.server = :webrick
```

Comment:
Why is this necessary? Can you point to the changelog please?

Reply:
@ChrisBr 

> Why is this necessary? Can you point to the changelog please?

In the changelog ( https://github.com/teamcapybara/capybara/blob/master/History.md ) it is not referred in version 3.1.0 to the change made in this commit: https://github.com/teamcapybara/capybara/commit/6f48859a607efcc4498e93d8026874af5afd4fb7

What should I put? A reference to a version that doesn't include the text that it have been changed to mandatory, or the number of the commit itself?

Reply:
But we're using webrick, do we? I'm wondering that this works?

https://github.com/openSUSE/open-build-service/blob/master/src/api/Gemfile#L133

Reply:
This work because we only use `unicorn-rails` on development. When we run test we are using `WEBrick`

Reply:
@DavidKang where is it required? I can not see where we configure Webbrick? I think we should use the same as we use in dev (production we use passenger)

Reply:
@ChrisBr, webrick comes by default with rails, right?

Reply:
I can not find it in the gemfile.lock so I guess not :thinking:

Reply:
@ChrisBr, cc: @DavidKang: `webrick` is installed with Ruby, as a standard library.

Since Rails 5, if not before, `puma` gem is used in Rails in development and test modes, but only if it is defined in the Gemfile. A default Gemfile of a new Rails application (at least from Rails 5) comes with `puma` as a dependency.

But, as we haven't introduced `puma` yet, we have to configure `capybara` manually  to use `webrick`.